core/app/models/schedule.rb

```diff
@@ -18,10 +19,33 @@ def self.from_sections sections
     end
   end
 
+  def findSortValues
```

Comment:
All ruby methods and variables should be `lower_snake_case`

Reply:
Okay thanks

Reply:
These should all be good now